src/encoded/static/components/navigation/NavigationBar.js

```diff
@@ -120,8 +120,8 @@ export class NavigationBar extends React.PureComponent {
 
     render() {
         const { testWarning, mobileDropdownOpen, mounted } = this.state;
-        const { href, context, schemas, browseBaseState, isFullscreen } = this.props;
-        const testWarningVisible = !!(testWarning & !isFullscreen); // Hidden on full screen mode.
+        const { href, context, schemas, browseBaseState, isFullscreen, currentAction } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['currentAction' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)